tokens/custom-spectrum/custom-vars.css

```
@@ -36,6 +36,9 @@ governing permissions and limitations under the License.
   --spectrum-animation-ease-out: cubic-bezier(0, 0, 0.40, 1);
   --spectrum-animation-ease-linear: cubic-bezier(0, 0, 1, 1);
 
+  --spectrum-corner-radius-1000-full: 9999px;
+  --spectrum-corner-radius-full: var(--spectrum-corner-radius-1000-full);
+  --spectrum-corner-radius-1000: var(--spectrum-corner-radius-1000-full);
```

```suggestion
--spectrum-corner-radius-1000: 9999px;
```
Thoughts on changing these three lines of changes into this single line? I noticed that the `--spectrum-corner-radius-full` alias is already set to the value of `--spectrum-corner-radius-1000` in the s2 tokens. And then we would shouldn't need a separate `--spectrum-corner-radius-1000-full` if we're using `--spectrum-corner-radius-full` in our components' CSS.

---

I think that makes sense. The cascade is going to be important here to ensure that `--spectrum-corner-radius-1000` is actually resolving to our `9999px` instead of the Spectrum Tokens value (that is, until we get a translation up and running to transform it for our needs).

---

Note for posterity: Patrick handled this in a tokens PR (#2597).